apps/builddao/widget/lib/everything-sdk.jsx

```diff
@@ -29,7 +29,10 @@ function fetchThings(app, type) {
 
   let things = [];
   flattenedKeys.forEach((key) =>
-    things.push(JSON.parse(Social.get(key, "final"))),
+    things.push({
```

Comment:
Social.get can take an array of keys (more performant)

JSON.parse(Social.get(flattenedKeys, "final"));

I think it should work the same way